src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -97,15 +97,37 @@ class ExplorerButtonGroup extends React.Component {
 
   getManifest = async (indexType) => {
     const refField = this.props.guppyConfig.manifestMapping.referenceIdFieldInDataIndex;
+    const md5Field = 'md5sum';
+    const fileNameField = 'file_name';
+    const fileSizeField = 'file_size';
+    const additionalFields = [md5Field, fileNameField, fileSizeField];
```

Comment:
trivial: since we are hard coding these anyways, seems a bit redundent to me for giving each of them a variable name and then put into array